nvidia.ps1

```diff
@@ -49,6 +49,13 @@ else {
 
 # Checking currently installed driver version
 Write-Host "Attempting to detect currently installed driver version..."
+if (Get-ItemProperty -Path HKLM:\SYSTEM\CurrentControlSet\Services\nvlddmkm -Name 'DCHUVen' -ErrorAction Ignore) {
+    Write-Host "DCH driver are not yet supported."
```

Comment:
Maybe add some more information. Maybe just a single line that windows will keep DCH driver up to date? Or just a link?